src/RA_Dlg_AchEditor.h

```diff
@@ -2,33 +2,25 @@
 #define RA_DLG_ACHEDITOR_H
 #pragma once
 
-#include "RA_httpthread.h"
 #include "RA_Achievement.h"
+#include "RA_httpthread.h"
 
 #include "ui\ImageReference.hh"
 
-namespace {
-const size_t MAX_CONDITIONS = 200;
-const size_t MEM_STRING_TEXT_LEN = 80;
-}
-
 class BadgeNames
 {
 public:
-    BadgeNames()
-        : m_hDestComboBox(nullptr)
-    {
-    }
     void InstallAchEditorCombo(HWND hCombo) { m_hDestComboBox = hCombo; }
 
     void FetchNewBadgeNamesThreaded();
     void AddNewBadgeName(const char* pStr, bool bAndSelect);
     void OnNewBadgeNames(const rapidjson::Document& data);
 
 private:
-    HWND m_hDestComboBox;
+    HWND m_hDestComboBox = nullptr;
 };
 
+enum class CondSubItems : std::size_t;
```

Comment:
This needs to be forward declared since it's not defined here.
If putting the enum as a private member is desired, CreateIPE needs to be a friend function.